docs/rackhd_vlab/web_ui.rst

```diff
@@ -1,2 +1,229 @@
 RackHD try-out with Web UI
 =============================
+
+In this session, you will learn how to configure web-ui and customize a RackHD workflow to implement your own logic.
+
+Configure on-web-ui
+~~~~~~~~~~~~~~~~~~~~
+
+* Open web browser, and then go to the following URL ``http://10.62.59.196:9090/ui``, replace with your own ipaddr and port.
+* Click the ``gear`` button on the top right panel
+
+.. image:: ../_static/web-ui/s1.png
+    :align: center
+
+* Enter your ``RackHD Northbound API``, then click ``save`` button
+
+.. image:: ../_static/web-ui/s2.png
+    :align: center
+
+* Then you will see all discovered nodes in the panel.
+
+.. image:: ../_static/web-ui/s3.png
+    :align: center
+
+Workflow scenario
+~~~~~~~~~~~~~~~~~~
+
+* You have a number of new bare metal servers coming online.
```

Comment:
You'd better remove this line. Here the condition is:
1. You have discovered the node.
2. You have configure OBM correctly.